boa_interner/src/lib.rs

```diff
@@ -180,178 +248,31 @@ impl Interner {
         self.resolve(symbol).expect("string disappeared")
     }
 
-    /// Gets the symbol of the static string if one of them
-    fn get_static(string: &str) -> Option<Sym> {
-        Self::STATIC_STRINGS
-            .into_iter()
-            .enumerate()
-            .find(|&(_i, s)| string == s)
-            .map(|(i, _str)| {
-                let raw = NonZeroUsize::new(i.wrapping_add(1)).expect("static array too big");
-                Sym::from_raw(raw)
+    /// Gets the symbol of the common string if one of them
+    fn get_common(string: &str) -> Option<Sym> {
+        COMMON_STRINGS.get_index(string).map(|idx|
+            // SAFETY: `idx >= 0`, since it's an `usize`,
+            // and `idx + 1 > 0` (considering no overflows, but
+            // an overflow would panic on debug anyways).
```

Comment:
Even if this would panic on debug, we should probably still use [`checked_add()`](https://doc.rust-lang.org/stable/std/primitive.usize.html#method.checked_add) to make sure it's still safe in release builds.

Reply:
Ah, I think I didn't explain well why we don't have to worry about overflows in this case